src/ActiveField.php

```diff
@@ -431,6 +431,8 @@ public function listBox($items, $options = [])
         if ($this->form->layout === ActiveForm::LAYOUT_INLINE) {
             Html::removeCssClass($this->labelOptions, 'visually-hidden');
         }
+        Html::removeCssClass($options, 'form-control');
+        Html::addCssClass($options, ['widget' => 'form-select']);
```

Comment:
This works but wouldn't be better to work on `$this->inputOptions`?

Reply:
@bizley can you post example?

Reply:
Hmm, I see there is no easy way to replace class. What about
```
Html::removeCssClass($this->inputOptions, ['widget' => 'form-control']);
Html::addCssClass($this->inputOptions, ['widget' => 'form-select']);
```

but I'm not quite sure if changing this property will not affect other possible calls for this widget.

Reply:
@bizley 1. it's `select`, not `input` 2. why this solution is better?

Reply:
Parent calls [this](https://github.com/yiisoft/yii2/blob/master/framework/widgets/ActiveField.php#L682)

Reply:
@bizley I'm know, but why this solution is better?

Reply:
Hmm, maybe you are right. Ok, let's do it like that here.